api/rest/restapi.go

```diff
@@ -258,6 +262,15 @@ func (api *API) addRoutes(router *mux.Router) {
 				),
 			)
 	}
+
+	statikFS, err := fs.New()
+	if err != nil {
+		panic(err)
+	}
+	staticServer := http.FileServer(statikFS)
+	sh := http.StripPrefix("/swaggerui/", staticServer)
+	router.PathPrefix("/swaggerui/").Handler(sh)
+
```

Comment:
This is different than other routes, but it seems like to make the logic similar(write a HandlerFunc), I will have to re-implement or copy things that are already done in http.FileServer.
Can we keep it like this? Anyway this this endpoint is much different than rest.
@lanzafame

Reply:
I would put it in a function `addSwaggerRoute(router *mux.Router)` and also it would be good to add the `ochttp` route tag:
```
router.PathPrefix("/swaggerui/").Handler(
    ochttp.WithRouteTag(
            http.HandlerFunc(sh),
	    "/swaggerui/",
    ),
)
```